modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/file-size-limit/FileSizeMimetypes.js

```diff
@@ -68,7 +92,13 @@ const FileSizeField = ({
 			<ClayInput
 				defaultValue={size}
 				id="size"
+				min="0"
```

Comment:
Do we have some kind of max value? May the max general uploads fit at maximum

Reply:
Not sure if we can get this value, but if we do, it may confuse the user why that maximum value, so we should inform him before by an alert info or something similar. I would treat this in a different story if necesarry. What do you think?

Reply:
Ok, good enough. 
But think about this, if the user adds a negative value, how are we validating this now?

Reply:
if the user adds a negative value, how are we validating this now?